styles.js

```diff
@@ -5,10 +5,24 @@ export default createGlobalStyle`
   *::before,
   *::after {
     box-sizing: border-box;
+    margin: 0;
+    padding: 0;
   }
 
   body {
-    margin: 0;
     font-family: system-ui;
+    margin: 1rem 0rem 0rem 1rem;
+  }
+
+  header {
+    margin-bottom: 2rem;
+  }
```

Comment:
Please move the styling to the header component. 🙏 

Please do this for the `ul` and `h1` selector as well.